### PR TITLE
Add initial documentation

### PR DESCRIPTION
**Pull request by richardmiller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+ to find its IP address.
+ 
+## Watch
+ 
+ The `watch` command will sync changes you make locally to a container that's part of the remote environment.
+ You need to specify which container to sync with using the docker composer service name. 
+ This will usually be the main application container. For example, if the service you want to sync to is web:
+  
+  ```
+  console watch web
```
Comment (richardmiller) [inline README.md]: This will need to change depending on the installation instructions

File: README.md [hunk 2]
```diff
@@ -1,3 +1,73 @@
 # Remote environment client
 
-This 
+A command line tool to help with using Continuous Pipe as a remote development environment.
+
+This helps to set up Kubectl, create, build and destroy remote environments and keep files
+in sync with the local filesystem.
+
+## Installation
+
```
Comment (sroze) [inline README.md]: The ideal to me would be that we create [GitHub releases](https://github.com/continuouspipe/remote-environment-client/releases) with an artifact being the bash script containing the Git tag as a comment in order to be able to see which version do we have. (or even a command would be fab tbh).

But yeah, simply `curl`ing ,`mv`ing and `chmod`ing the bash script sounds good to me, nah?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,73 @@
...
+## Create
+
+The `create` command will push changes the branch you have checked out locally to your remote 
+ environment branch. Continuous Pipe will then build the environment. YOu can use the [Continuous Pipe admin
+ site](http://ui.continuouspipe.io/) to see when the environment has finished building and 
+ to find its IP address.
+ 
+## Watch
+ 
+ The `watch` command will sync changes you make locally to a container that's part of the remote environment.
```
Comment (sroze) [inline README.md]: Maybe that before:

**Note:** you need to wait the remote environment to be created before running this command.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,73 @@
...
+
+This helps to set up Kubectl, create, build and destroy remote environments and keep files
+in sync with the local filesystem.
+
+## Installation
+
+
+## Setup
+
+To start using this tool for a project, run the `setup` command from the project root.
```
Comment (sroze) [inline README.md]: I would provide just an example to make it clear to fast readers:
```
$ cp-remote setup
```

Comment (tonypiper) [inline README.md]: should this be renamed from `console` to `cp-remote` then?

**Comment by sroze** [comment]
Amazing work @richardmiller ! 👍

**Pull request merged**